aiochsa/pool.py

```diff
@@ -14,22 +14,34 @@ def dsn_to_params(dsn):
             f'Expecting "clickhouse" scheme in DSN, got {parsed.scheme}'
         )
 
-    hostname = parsed.hostname or '127.0.0.1'
-    port = parsed.port or 8123
+    # Parts to unquote PCT-encoded are:
+    #   - host
+    #   - userinfo parts (username, password)
+    #   - path
+    # https://datatracker.ietf.org/doc/html/rfc3986
+
+    hostname = unquote(parsed.hostname) or '127.0.0.1'
```

Comment:
I get why `unquote()` is used for other parts.
But i cannot imagine quoted hostname.
Have you ever met one?

I know this will not danage the data.
It just confuses me.

Reply:
I doubt it will be ever used for hostname. But the RFC clearly states that it _can_ be used, so we must unquote. Also I see that [asyncpg does this](https://github.com/MagicStack/asyncpg/blob/master/asyncpg/connect_utils.py#L227).